.github/test-spec.yml

```diff
@@ -0,0 +1,25 @@
+# This is the Jenkins ci variant of the .github/labler.yaml
+
+CI-ble-test:
+  - 'softdevice_controller/**/*'
```

Comment:
Can we update the path to:
'softdevice_controller/include/**/*'
'softdevice_controller/lib/**/*'

The doc update don't need to trigger the downstream CI

Same for the path under other test labels

Reply:
Sure